frontend/src/components/FooTouer/FooTouer.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './FooTouer.css';
+
+function FooTouer() {
+  return (
+    <div className="FooTouer">
+      HelloWorld
+    </div>
+  );
+}
+
+export default FooTouer;
```

Comment:
Dlaczego nie App.tsx, ?

Reply:
To samo co wyżej, jest taka konwencja, aby główny komponent nazywał się tak jak apka. Dlatego wszystkie App zastąpiłam FooTouer